README.md

```diff
@@ -25,6 +25,7 @@ yarn install
 - ⤵️ 🔹 [Withdraw Ether](./packages/eth-withdraw/)
 - ⤴️ 💸 [Deposit Token](./packages/token-deposit/)
 - ⤵️ 💸 [Withdraw token](./packages/token-withdraw/)
+- ⤴️ 🔹 [Contract deposit](./packages/contract-deposit/)
```

Comment:
Should we use something a bit more descriptive like "L2 Alias Control and Fund Transfer Guide"?